src/GPL-1.0.xml

```diff
@@ -0,0 +1,240 @@
+<SPDX name="GNU General Public License v1.0 only" identifier="GPL-1.0" osi-approved="false">
```

Comment:
This [still][1] has ` only`, which I don't think we want.

[1]: https://github.com/spdx/license-list-XML/pull/83/#discussion_r102152377